.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
```

Comment:
@kategarklavs thanks!

@rogeruiz I think what was throwing me off is the "[Website] - [UI components]" both being together on that line. I was thinking that I needed to replace "website" with some website, and "UI components" with the component I opened this about. If it were just one it would make more sense, bc I think that's what we intend people to do. Maybe it should be more generalized? [name of issue] OR just use [UI component] bc I think that's more likely and more important than the website itself.